src/EnergyPlus/ZoneContaminantPredictorCorrector.cc

```diff
@@ -2503,11 +2503,11 @@ void CorrectZoneContaminants(EnergyPlusData &state,
                 if (state.dataContaminantBalance->Contaminant.CO2Simulation) {
                     DownInterpolate4HistoryValues(PriorTimeStep,
                                                   state.dataHVACGlobal->TimeStepSys,
-                                                  state.dataContaminantBalance->ZoneAirCO2(ZoneNum),
                                                   state.dataContaminantBalance->CO2ZoneTimeMinus1(ZoneNum),
                                                   state.dataContaminantBalance->CO2ZoneTimeMinus2(ZoneNum),
                                                   state.dataContaminantBalance->CO2ZoneTimeMinus3(ZoneNum),
                                                   state.dataContaminantBalance->CO2ZoneTimeMinus4(ZoneNum),
+                                                  state.dataContaminantBalance->CO2ZoneTimeMinus4(ZoneNum),
                                                   state.dataContaminantBalance->ZoneAirCO2(ZoneNum),
                                                   state.dataContaminantBalance->DSCO2ZoneTimeMinus1(ZoneNum),
                                                   state.dataContaminantBalance->DSCO2ZoneTimeMinus2(ZoneNum),
```

Comment:
As an example of what this change represents. In this case, if the simulation down-shifted, the value of ZoneAirCO2 would be set equal to ZoneAirCO2 and would not get reverted to what it was at the last time step. In addition to that, the new history terms would be referenced from the value of ZoneAirCO2 at the end of the time step that would be resimulated at shorter time steps (i.e., the longer simulation results should be deleted/reverted and not used). This should not cause noticable differences in results but there would certainly be numeric differences from the correct simulation results.